tree-sitter.json

```diff
@@ -0,0 +1,36 @@
+{
+  "grammars": [
+    {
+      "name": "sway",
+      "camelcase": "Sway",
+      "scope": "source.sway",
+      "path": ".",
+      "file-types": [
+        "sw"
+      ],
+      "injection-regex": "sway"
+    }
+  ],
+  "metadata": {
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "description": "Sway grammar for tree-sitter",
+    "authors": [
+      {
+        "name": "Fuel Labs",
+        "email": "contact@fuel.sh"
+      }
+    ],
+    "links": {
+      "repository": "git+https://github.com/FuelLabs/tree-sitter-sway.git"
+    }
+  },
+  "bindings": {
+    "c": true,
+    "go": true,
+    "node": true,
+    "python": true,
+    "rust": true,
+    "swift": true
+  }
```

Comment:
Shouldn't the rust binding be the only one set to true?

Reply:
And `node`, I'd say.

Reply:
It looks like `tree-sitter generate` should generate these bindings. Perhaps we should leave them set to true and also check in the generated bindings.

Reply:
Yes, it does; but people often complain (understandably) if a bugfix PR comes with a metric butt-ton of weird files and bindings ;) So I imagine this PR intentionally omitted most "optional" files (even though one or two related changes slipped through).